scripts/build.sh

```diff
@@ -41,6 +65,14 @@ while [[ $# -gt 0 ]] ; do
 			# call this script with -build-deps
 			BUILD_DEPS="1"
 			;;
+		-build-prefix)
+			# instead of building in build & build-mobile in the current directory, build in <buildprefix>build
+			# and <buildprefix>build-mobile; notice that there's no slash between the prefix and the two directory
+			# names, so if the prefix is supposed to be a path, add the slash at the end of it, or do funky things
+			# where build/build-mobile get appended to partial path name
+			shift
+			BUILD_PREFIX="$1"
```

Comment:
No reason to quote $1 here, No globbing happens on assignment.

Reply:
I am not a connoisseur of $shell. I believe you :-)